pkg/catalogv2/helm/release.go

```diff
@@ -111,6 +117,7 @@ func getReleaseDataAndKind(obj runtime.Object) (string, error) {
 	return "", ErrNotHelmRelease
 }
 
+// resourcesFromManifest returns the list of v1.ReleaseResource present in the given manifest
```

Comment:
// resourcesFromManifest uses the wrangler functions to convert the rendered manifest template
// to a list of runtime.Objects and then convert it to v1.ReleaseResouce list to return it.